CondFormats/JetMETObjects/BuildFile.xml

```diff
@@ -2,6 +2,7 @@
 <use   name="FWCore/ParameterSet"/>
 <use   name="FWCore/Utilities"/>
 <use   name="CondFormats/Serialization"/>
+<use   name="CondFormats/DataRecord"/>
```

Comment:
spurious dependency: if you remove the includes as in https://github.com/cms-sw/cmssw/pull/10350/files#r35481619 this is not needed anymore.